HelpSource/Classes/ArrayedCollection.schelp

```diff
@@ -3,7 +3,7 @@ categories::Collections>Ordered
 summary:: Abstract superclass of Collections of fixed maximum size
 
 DESCRIPTION::
-ArrayedCollection is an abstract class, a subclass of SequenceableCollections whose elements are held in a vector of slots. Instances of ArrayedCollection have a fixed maximum size beyond which they may not grow.
+link::Classes/ArrayedCollection:: is an abstract class, a subclass of link::Classes/SequenceableCollection:: whose items are held in a vector of slots, indexable by a link::Classes/SimpleNumber:: starting from 0. If a float is used as index, it will be truncated. Instances of link::Classes/ArrayedCollection:: have a fixed maximum size beyond which they may not grow.
```

Comment:
You don't need to `link::` to the class of the current help file, you can just used `code::ArrayedCollection::`, or `teletype::`

Reply:
Previously, I used 'code::ClassName::' or 'teletype::ClassName::' in this case (the class name is the name of the current help file).

However, a reviewer pointed this out to me once. Since then, I have always used 'link::Classes/classname::'.
I agree with your opinion, but what is the official way to do it?

Reply:
I left it as it is, I think there is no reason to use `link::Classes/ClassName:: when the the class name is the name of the current help file. 

However, I will follow the standard.

I could not edit [the document @mtmccrea  mentioned](https://github.com/supercollider/supercollider/wiki/Style-Guidelines%3A-SCDocs).